src/ensenso_driver.cpp

```diff
@@ -43,13 +45,15 @@ class EnsensoDriver
     image_transport::CameraPublisher  r_raw_pub_;
     image_transport::Publisher        l_rectified_pub_;
     image_transport::Publisher        r_rectified_pub_;
+    ros::Publisher                    disparity_pub_;
```

Comment:
Hmm, I suppose with the other images is probably fine.  It seems a little odd to keep the message publishers grouped separately from the other publishers, but might as well stick with the original style.